### DISCUSSION TITLE
Merge Palettes option

### DISCUSSION THREAD

**Discussion by Ryder17z** [Orama-Interactive/Pixelorama#500] [Ideas]
I'd like to merge two palettes.

I could maybe do it by hand if there was an actual export option but i haven't found one.

**Comment by novhack** [comment]
Interesting idea and it for sure makes sense to have this feature.

I have two ideas how it could be done:
1. When creating a new palette have an option to select multiple palettes to generate it from. Currently in v0.9 it's possible to copy colors from only one currently selected palette when creating a new palette.
2. Add this feature as a button to side panel where other palettes features will be - in v0.9 there are currently only buttons to add/remove colors. When button is pressed a selector could appear to select a palette from which to copy all colors to current palette.

I would chose only one option to avoid having multiple workflow paths for the same effect.

**Reply by Ryder17z** [reply]
If we can have multi-select on number 2, that I think would be great because it is more flexible than no.1 but can easily perform the same action.

I suggest a checkbox approach with a separate "add to current palette" button. (this would benefit from an "invert selection" button as well, because if you have selected none and press, it, you get the functionality of a "select all" button also)
